toolchest_client/tools/tool.py

```diff
@@ -180,16 +182,14 @@ def _validate_args(self):
 
         if self.inputs is None:
             raise ValueError("No input provided.")
-        if self.output_path is None:
-            raise ValueError("No output path provided.")
-        if not os.access(
+        if self.output_path and not os.access(
                 os.path.dirname(self.output_path),
                 os.W_OK | os.X_OK,
         ):
             raise OSError("Output file path must be writable.")
         if not self.output_name:
             raise ValueError("Output name must be non-empty.")
-        if self.output_is_directory and not os.path.isdir(self.output_path):
+        if self.output_is_directory and self.output_path and not os.path.isdir(self.output_path):
```

Comment:
Out of curiosity, why was this rewritten?

Reply:
This was rewritten to allow for `output_path=None`, which skips the local download.